fireapp/lib/domain/models/new/qualification.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:flutter/foundation.dart';
+
+part 'qualification.freezed.dart';
+part 'qualification.g.dart';
+
+@freezed
+class Qualification with _$Qualification {
+  const factory Qualification({
+    required int id,
+    required String name,
+    required bool deleted,
+    @JsonKey(name: 'last_update_datetime') DateTime? updateDateTime,
```

Comment:
I'd prefer for the names to be consistent (i.e, this called lastUpdateDateTime, the next line called createdDateTime)

Reply:
Thank you! I will change these